tf-ncl/src/nickel_builder.rs

```diff
@@ -156,31 +164,49 @@ impl Field<Record> {
     pub fn no_value(mut self) -> Record {
         self.record.fields.push((
             self.path,
-            Term::MetaValue(self.metadata.unwrap_or_default()).into(),
+            record::Field {
+                value: None,
```

Comment:
```suggestion
```

Because `None` is the default of `Option`, you're not required to set it explicitly.